### ISSUE TITLE
the loss up down and up down

### ISSUE THREAD

**Issue by beihaidao** [facebook/C3D#303]
when i run :finetuning_ucf101.sh in v11, then i get the result like this:
I0828 19:37:39.324069 13760 solver.cpp:238]     Train net output #0: ## loss = 0.0293793 (* 1 = 0.0293793 loss)
I0828 19:37:39.324079 13760 sgd_solver.cpp:105] Iteration 16000, lr = 1e-06
I0828 19:40:36.826422 13760 solver.cpp:219] Iteration 16200 (1.12675 iter/s, 177.502s/200 iters),##  loss = 0.0536419
I0828 19:40:36.827183 13760 solver.cpp:238]     Train net output #0:##  loss = 0.0536413 (* 1 = 0.0536413 loss)
I0828 19:40:36.827193 13760 sgd_solver.cpp:105] Iteration 16200, lr = 1e-06
I0828 19:43:34.383196 13760 solver.cpp:219] Iteration 16400 (1.12641 iter/s, 177.555s/200 iters),##  loss = 1.25873
I0828 19:43:34.383977 13760 solver.cpp:238]     Train net output #0: ## loss = 1.25873 (* 1 = 1.25873 loss)
I0828 19:43:34.383991 13760 sgd_solver.cpp:105] Iteration 16400, lr = 1e-06
I0828 19:46:31.802453 13760 solver.cpp:219] Iteration 16600 (1.12728 iter/s, 177.418s/200 iters), loss = 1.23278
I0828 19:46:31.803254 13760 solver.cpp:238]     Train net output #0: ## loss = 1.23278 (* 1 = 1.23278 loss)
I0828 19:46:31.803267 13760 sgd_solver.cpp:105] Iteration 16600, lr = 1e-06
I0828 19:49:29.236555 13760 solver.cpp:219] Iteration 16800 (1.12719 iter/s, 177.433s/200 iters), ## loss = 0.495886
I0828 19:49:29.237337 13760 solver.cpp:238]     Train net output #0: loss = 0.495885 (* 1 = 0.495885 loss)
I0828 19:49:29.237350 13760

anyone knows how to solve this problem?

**Comment by dutran** [comment]
you can reduce lr or stop training. Is this UCf101?

**Comment by beihaidao** [comment]
@dutran yes ,is the ucf101,iwill try to reduce the lr.

**Comment by titikid** [comment]
i wonder how this loss calculated? for all training samples or only small part (mini-batch) of them?
